### PR TITLE
[IA-5056] Update TNU version

### PR DESCRIPTION
**Pull request by LizBaldo**
__Jira ticket__: https://broadworkbench.atlassian.net/browse/IA-5056

### What

- Bumps up versions of all terra-docker images that were built as part of https://github.com/DataBiosphere/terra-docker/pull/495
- Increases the size of the Dataproc Master disk from 120GB to 150GB to accommodate for the bigger google tensorflow base image that we are using as our parent image, see https://console.cloud.google.com/artifacts/docker/deeplearning-platform-release/us/gcr.io/tf-gpu.2-11.py310
- There was no need to increase the size of the GCE boot disk because the hail image is not cached in there so we have a bit more buffer room 

### Why

- TNU released a new version, and since this package is installed in the base image, all children images needed to be regenerated as well

### What to test

- 

### Who tested and where

- [ ] This change is covered by automated tests 
  - _NB: Rerun automation tests on this PR by commenting `jenkins retest` or `jenkins multi-test`._
- [ ] I validated this change 
- [ ] Primary reviewer validated this change 
- [ ] I validated this change in the dev environment

### CODE REVIEW

File: jenkins/dataproc-custom-images/prepare-custom-leonardo-jupyter-dataproc-image.sh [hunk 1]
```diff
@@ -16,18 +16,18 @@ set -e -x
...
+terra_jupyter_hail="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-hail:1.1.13"
+terra_jupyter_gatk="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:2.3.8"
+terra_jupyter_aou="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:2.2.14"
 welder_server="us.gcr.io/broad-dsp-gcr-public/welder-server:8667bfe"
 openidc_proxy="broadinstitute/openidc-proxy:2.3.1_2"
 anvil_rstudio_bioconductor="us.gcr.io/broad-dsp-gcr-public/anvil-rstudio-bioconductor:3.19.0"
 
 # Note that this is the version used currently by AOU in production, the one above can be staged for testing
-terra_jupyter_aou_old="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:2.2.7"
+terra_jupyter_aou_old="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:2.2.13"
```
Comment (lucymcnatt) [inline jenkins/dataproc-custom-images/prepare-custom-leonardo-jupyter-dataproc-image.sh:30]: is AOU ready for this update?

Comment (LizBaldo) [reply jenkins/dataproc-custom-images/prepare-custom-leonardo-jupyter-dataproc-image.sh:30]: Good question, sorry I should have left a breadcrumb, but AOU has been using 2.13 in prod for a few months now, so this should not change anything and we can tell them to start testing for 2.14 in pre-prod. 2.2.7 was just gathering dust for nothing

Comment (LizBaldo) [reply jenkins/dataproc-custom-images/prepare-custom-leonardo-jupyter-dataproc-image.sh:30]: @Qi77Qi could you confirm that our pro and pre-prod versions of the AOU image are correct?

**Review by samanehsan — APPROVED** [review]

**Review by lucymcnatt — APPROVED** [review]

**Review by cindy-broadinstitute — APPROVED** [review]

File: jenkins/dataproc-custom-images/prepare-custom-leonardo-jupyter-dataproc-image.sh (hunk 1 above)
Comment (LizBaldo) [reply jenkins/dataproc-custom-images/prepare-custom-leonardo-jupyter-dataproc-image.sh:30]: @lucymcnatt confirming that AOU prod is on 2.13 already: https://github.com/all-of-us/workbench/blob/7f4639e9334bbc3526c2a7676b13d585b23cb541/api/config/config_prod.json#L15

**Pull request merged**